planetscale/dbutil/dbutil_test.go

```diff
@@ -172,3 +174,13 @@ func parseCert(pemCert string) (*x509.Certificate, error) {
 	}
 	return x509.ParseCertificate(bl.Bytes)
 }
+
+func Test_generateKey(t *testing.T) {
```

Comment:
Can you update this to use camel casing? So `TestGenerateKey`? 

I think this test can also be updated to assert that the default key type generated is always ECDSA and that the RSA parts can be dropped here.